tests/pytests/integration/modules/test_file.py

```diff
@@ -0,0 +1,162 @@
+"""
+Tests for the file state
+"""
+import pytest
+import salt.states.file
+from tests.support.helpers import slowTest
+
+
+@pytest.mark.parametrize("verify_ssl", [True, False])
+@slowTest
+def test_get_source_sum_verify_ssl_false(
+    salt_call_cli, tmpdir, ssl_webserver, verify_ssl
+):
+    """
+    test verify_ssl with get_source_sum
+    """
+    web_file = ssl_webserver.url("this.txt")
+    ret = salt_call_cli.run(
+        "--local",
+        "file.get_source_sum",
+        tmpdir.join("test_source_sum.txt").strpath,
+        web_file,
+        web_file + ".sha256",
+        "verify_ssl={}".format(verify_ssl),
+    )
+    if not verify_ssl:
+        assert (
+            ret.json["hsum"]
+            == "f2ca1bb6c7e907d06dafe4687e579fce76b37e4e93b7605022da52e6ccc26fd2"
+        )
+        assert ret.json["hash_type"] == "sha256"
+    else:
+        assert "SSL: CERTIFICATE_VERIFY_FAILED" in ret.stderr
+
+
+@pytest.mark.parametrize("verify_ssl", [True, False])
+@slowTest
+def test_get_managed_verify_ssl(salt_call_cli, tmpdir, ssl_webserver, verify_ssl):
+    """
+    test verify_ssl with get_managed
+    """
+    web_file = ssl_webserver.url("this.txt")
+    ret = salt_call_cli.run(
+        "--local",
+        "file.get_managed",
+        tmpdir.join("test_managed.txt").strpath,
+        "",
+        web_file,
+        web_file + ".sha256",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "base",
+        "{}",
+        "",
+        "True",
+        "verify_ssl={}".format(verify_ssl),
+    )
```

Comment:
FYI,
If you pass these as keyword arguments to `run()` you would likely not need all of the `"",` in the call.
```
ret = salt_call_cli.run(
    "--local",
    "file.get_managed",
    tmpdir.join("test_managed.txt").strpath,
   saltenv="base"
```
Bah!
Nevermind, it's all arguments to the function, not keyword arguments.
_Bad code, bad bad code...._
Anyway, you can still do ` .run(... args..., verify_ssl=True)` , no need to pass `verify_ssl=True` as a string argument.